frontends/qt/main.cpp

```diff
@@ -124,6 +127,21 @@ class RequestInterceptor : public QWebEngineUrlRequestInterceptor {
             return;
         }
 
+        // All the requests originated in the wallet-connect section are allowed, as they are needed to
+        // load the Dapp logos and it is not easy to filter out non-images requests.
+        bool onWCPage = currentUrl.contains(QRegularExpression("^qrc:/account/[^\/]+/wallet-connect/.*$"));
+        if (onWCPage) {
+          info.block(false);
+          return;
+        }
+
+        // Needed for the wallet connect workflow.
+        bool VerifyWCRequest = requestedUrl.contains(QRegularExpression("^https://verify.walletconnect.com/.*$"));
```

Comment:
use `\.` for dots, otherwise verify-walletconnect.com etc. is also accepted

Reply:
ouch! good catch, thanks